WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/HelpAndSupportScreen.java

```diff
@@ -0,0 +1,60 @@
+package org.wordpress.android.e2e.pages;
+
+import androidx.test.espresso.ViewInteraction;
+import androidx.test.espresso.action.ViewActions;
+
+import org.wordpress.android.R;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static org.wordpress.android.support.WPSupportUtils.populateTextField;
+import static org.hamcrest.Matchers.anyOf;
+import static org.wordpress.android.support.WPSupportUtils.waitForElementToBeDisplayedWithoutFailure;
+
+
+public class HelpAndSupportScreen {
+    static ViewInteraction contactUsButton = onView(withId(R.id.contact_us_button));
+    static ViewInteraction faqButton = onView(withId(R.id.faq_button));
+    static ViewInteraction myTicketsButton = onView(withId(R.id.my_tickets_button));
+    static ViewInteraction applicationLogButton = onView(withId(R.id.application_log_button));
+    static ViewInteraction applicationVersionText = onView(withId(R.id.applicationVersion));
+    static ViewInteraction emailAddressText = onView(withId(R.id.contactEmailAddress));
+
+
+    public HelpAndSupportScreen assertHelpAndSupportScreenLoaded() {
+        contactUsButton.check(matches(isCompletelyDisplayed()));
+        faqButton.check(matches(isCompletelyDisplayed()));
+        myTicketsButton.check(matches(isCompletelyDisplayed()));
+        applicationLogButton.check(matches(isCompletelyDisplayed()));
+        applicationVersionText.check(matches(isCompletelyDisplayed()));
+        emailAddressText.check(matches(isCompletelyDisplayed()));
+        return this;
+    }
+
+    public ContactSupportScreen openContactUs() {
+        contactUsButton.perform(ViewActions.click());
+        setEmailIfNeeded("WPcomTest@test.com", "TestUser");
+        return new ContactSupportScreen();
+    }
+
+    public void setEmailIfNeeded(String emailAddress, String userName) {
+        ViewInteraction emailInput = onView(withId(R.id.support_identity_input_dialog_email_edit_text));
+
+        if (!waitForElementToBeDisplayedWithoutFailure(emailInput)) {
+            return;
+        }
+
+        populateTextField(emailInput, emailAddress);
+        ViewInteraction nameInput = onView(withId(R.id.support_identity_input_dialog_name_edit_text));
+        populateTextField(nameInput, userName);
+
+        onView(anyOf(
+                withText(android.R.string.ok),
+                withId(android.R.id.button1)
+        ))
```

Comment:
This is the locator for `OK` button in this dialog:

<img width="321" alt="Screenshot 2021-12-23 at 21 35 35" src="https://user-images.githubusercontent.com/73365754/147284861-13cf2d85-0cc5-4269-8d53-f68a1a9df484.png">

I used something like `onView(withText("OK")` initially, and this worked locally, but never on CI. Thanks to [this discussion](https://www.py4u.net/discuss/692162), I found out that "OK" button might be presented differently by different Android versions, while `android.R.id.button1` always stands for the "positive" option in the dialog.